src/services/transform.ts

```diff
@@ -16,6 +19,63 @@ export default (config: ConfigManager): TransformService => {
         throw new Error('Failed to transform the imported xml: ' + e.message)
       }
       return response.text()
+    },
+    articleMetaOrderTransform: async (xml: string) => {
+      const documentNode = new DOMParser().parseFromString(xml, 'text/xml');
+      const result = await fontoxpath.evaluateUpdatingExpression(articleMetaOrderingExpression, documentNode);
+      fontoxpath.executePendingUpdateList(result.pendingUpdateList);
+      return new XMLSerializer().serializeToString(documentNode)
     }
   }
-}
\ No newline at end of file
+}
+
+const articleMetaOrderingExpression = `
```

Comment:
Something to follow up on later, but might want to add an issue to capture the work, is that I can see this being useful to others perhaps if ordering is as important to them as it is to us. As such, they may want use the same transformer but supply a different config hence moving this out to another file, or having it loaded at start would be good for reuse.
Nothing to worry about now though, this will be fine for MVP.